src/views/Detail/pages/DetailPage.tsx

```diff
@@ -75,6 +76,10 @@ const DetailPage = () => {
     fetchData();
   }, []);
 
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [selectedTab]);
+
```

Comment:
어차피 selectedTab이 변경될 때마다 컴포넌트가 리렌더링 되기 때문에
```suggestion
  useEffect(() => {
    window.scrollTo(0, 0);
  });

```

이렇게 사용해주면 되겠네용 ㅎㅎ